bc_obps/registration/api/v1/_user_operators/current.py

```diff
@@ -1,20 +1,25 @@
-from registration.constants import USER_OPERATOR_TAGS
+from registration.schema.v1.user_operator import PendingUserOperatorOut
 from service.data_access_service.user_service import UserDataAccessService
+from registration.constants import USER_OPERATOR_TAGS
+from registration.api.utils.current_user_utils import get_current_user_guid
 from registration.decorators import authorize, handle_http_errors
-from registration.schema.v1 import PendingUserOperatorOut
 from registration.schema.generic import Message
 from registration.api.router import router
 
 from registration.models import (
     UserOperator,
 )
 from service.error_service.custom_codes_4xx import custom_codes_4xx
-from registration.api.utils.current_user_utils import get_current_user_guid
+
+# We don't need a user_operator_id parameter (using "current" instead) because we can look up the user_operator via the user data we receive in the middleware
 
 
+# GET
+# This endpoint retrieves data about both the user-operator and the operator.
 @router.get(
```

Comment:
`/user-operator/user-operator-from-user` was similar to `user-operator/user-operator-operator`, so I've combined them into `/user-operators/current`. This means that `cas-registration/bciers/apps/registration/app/(authenticated)/dashboard/page.tsx` now receives some additional info from the API (legal name and isNew)